cmd/limactl/main_darwin.go

```diff
@@ -0,0 +1,9 @@
+//go:build darwin && !no_vz
```

Comment:
I like to avoid double-negation, when possible:

```suggestion
//go:build darwin && driver_vz
```

Then the `Makefile` will just have a `DRIVER_TAGS=driver_qemu,driver_vz,driver_wsl` setting that the user can override to only include the desired driver(s).